taxi_service/settings.py

```diff
@@ -43,6 +43,8 @@
     "django.contrib.messages",
     "django.contrib.staticfiles",
     "debug_toolbar",
+    # "crispy_bootstrap4",
```

Comment:
wow, i don't know how you managed to make crispy work with bootstrap like that :) it only worked for me when i added this to apps and installed crispy-bootstrap4 additionally...

Reply:
I don't know why, but if I remove the comment, my project doesn't work and django writes an error that there is no such module. But I installed it